trytond/res/user.py

```diff
@@ -256,6 +258,11 @@ def validate_password(cls, password, users):
 
     @staticmethod
     def get_sessions(users, name):
+        # AKE: manage session on redis
+        if security.config_session_redis():
+            dbname = Pool().database_name
+            return dict((u.id, security.redis.count_sessions(dbname, u.id))
```

Comment:
Soo  python 2.6...
return {ud.id: security.redis.count_sessions(dbname, u.id) for u in users}